packages/core/src/actor/instance.ts

```diff
@@ -601,13 +609,60 @@ export class ActorInstance<
 					driver,
 					this.#connStateEnabled,
 				);
-				this.#connections.set(conn.id, conn);
 
-				// Register event subscriptions
-				for (const sub of connPersist.su) {
-					this.#addSubscription(sub.n, conn, true);
-				}
+				// send ping, to ensure the connection is alive
+
+				const { promise, resolve } = Promise.withResolvers<void>();
+				restorePromises.push(
+					Promise.race([
+						promise,
+						new Promise<void>((_, reject) => {
+							setTimeout(() => {
+								reject();
+							}, 2500);
+						}),
+					])
+						.catch(() => {
+							process.nextTick(() => {
```

Comment:
is this portable? i think we need to use setTimeout(x, 0) instead

need comment on why this is running in the next tick